.github/actions/e2e-quantic-setup/action.yml

```diff
@@ -12,8 +12,8 @@ runs:
   steps:
     - run: |
         echo "${{ inputs.jwtkey }}" > server.key
-        npx --no-install ts-node scripts/build/deploy-community.ts --ci --scratch-org-def-path=./config/lws-disabled-scratch-def.json
-        npx --no-install ts-node scripts/build/deploy-community.ts --ci --scratch-org-def-path=./config/lws-enabled-scratch-def.json
+        pnpm exec ts-node scripts/build/deploy-community.ts --ci --scratch-org-def-path=./config/lws-disabled-scratch-def.json
```

Comment:
`--no-install` also exists for `pnpm exec` is there a reason why we don't change it 1 for 1?